File: Data Structures & Algorithms/kth-largest-integer-in-a-stream/submission-0.java
class KthLargest {
    int k; 
    ArrayList<Integer>arr; 


    public KthLargest(int k, int[] nums) {
        this.k=k; 
        arr=new ArrayList<>();
        for(int num:nums){
            arr.add(num);
        }
        
    }
    
    public int add(int val) {
        arr.add(val);
        Collections.sort(arr);
        return arr.get(arr.size()-k);
        
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        #given an integer nums an intrger K 
        #return k most fre element within array 

        #return output in any order 

        #so this way to create a dictionary that
        #has key and how many time each key appaea in the given aray 
        seen={}
        
        for num in nums:
            seen[num]=1+seen.get(num, 0) 

        array=[]
        for num, count in seen.items():
            array.append([count, num])
        array.sort()#now the array is sorted 

        result=[]
        while len(result)<k:
            result.append(array.pop()[1])
        return result 


